src/main/java/com/meilisearch/sdk/IndexesHandler.java

```diff
@@ -67,7 +72,33 @@ String get(String uid) throws MeilisearchException {
      * @return Meilisearch API response
      * @throws MeilisearchException if an error occurs
      */
-    String getAll() throws MeilisearchException {
+    Results<Index> getIndexes() throws MeilisearchException {
+        return httpClient.get("/indexes", Results.class, Index.class);
+    }
+
+    /**
+     * Gets indexes in the current Meilisearch instance
+     *
+     * @param param accept by the indexes route
+     * @return Meilisearch API response
+     * @throws MeilisearchException if an error occurs
+     */
+    Results<Index> getIndexes(IndexesQuery param) throws MeilisearchException {
+        URLBuilder urlb = new URLBuilder();
+        urlb.addSubroute("indexes")
```

Comment:
I pointed in the other PRs a similar approach where we introduce this code inside of the query handler :)